src/sentry/tsdb/redis.py

```diff
@@ -668,6 +684,7 @@ def record_frequency_multi(self, requests, timestamp=None, environment_id=None):
                     # Figure out all of the keys we need to be incrementing, as
                     # well as their expiration policies.
                     for rollup, max_values in self.rollups.items():
+                        chunk = []
```

Comment:
pyright complained about this being undefined. I suspect there might actually be a bug here, because we iterate `chunk` later on, but it will only be the chunk from the last `environment_id`.

Not going to attempt to improve since we don't really use redis tsdb much any more.